packages/serverless/src/integrations/aws_lambda.ts

```diff
@@ -0,0 +1,248 @@
+      });
+    }
+
+    /**
+     * This function sets tags which are displayed in Sentry Dashboard
+     * @param scope  - holds additional event information
+     * @hidden
+     */
+    function setTags(scope: Scope): void {
+      scope.setTag('runtime', `${nodeJs} ${global.process.version}`);
+      scope.setTag('transaction', setupOnceContext.awsContext.functionName);
+      scope.setTag('runtime.name', nodeJs);
+      scope.setTag('server_name', String(process.env._AWS_XRAY_DAEMON_ADDRESS));
+      scope.setTag('url', 'awslambda:///' + setupOnceContext.awsContext.functionName);
+    }
+
+    /**
+     * This function return event within hub
+     * @param eventId  - contains event Id of the event
+     * @hidden
+     */
+    function invokeEvent(eventId: String): void {
+      if (hub && hub.getIntegration(AWSLambda)) {
+        hub.withScope((scope: Scope) => {
+          scope.addEventProcessor(event => {
+            event.event_id = eventId.toString();
+            return event;
+          });
+        });
+      }
+    }
+
+    /**
+     * setting parameters in scope which will be displayed as additional data in Sentry dashboard
+     * @hidden
+     */
+    function setParameters(): void {
+      if (hub && hub.getIntegration(AWSLambda)) {
+        // setting parameters in scope which will be displayed as additional data in Sentry dashboard
+        hub.configureScope((scope: Scope) => {
+          setTags(scope);
+          // runtime
+          setAdditionalRuntimeData(scope);
+          // setting the lambda parameters
+          setAdditionalLambdaParameters(scope);
+          // setting the cloudwatch logs parameter
+          setCloudwatchLogsData(scope);
+          // setting the sys.argv parameter
+          scope.setExtra('sys.argv', process.argv);
+        });
+      }
+    }
+
+    /**
+     * unhandled and handled exception
+     * @param error  - holds the error captured in AWS Lambda Function
+     * @param id  - holds event id value
+     * @param callback  - callback function
+     */
+    rapidRuntime.prototype.postInvocationError = function(error: Error, id: string, callback: () => void): void {
+      /** return current event with event id  */
+      invokeEvent(id);
+
+      /**
+       * setting parameters in scope which will be displayed as additional data in Sentry dashboard
+       */
+      setParameters();
+
+      /** capturing the exception and re-directing it to the Sentry Dashboard */
+      hub.captureException(error);
+
+      /**
+       * Here, we make sure the error has been captured by Sentry Dashboard
+       * and then re-raised the exception
+       */
+      setTimeout(() => {
+        // re-raising the exception so as to capture it as an event.
+        invokedFunction.call(this, error, id, callback);
+      }, 1000);
```

Comment:
Why call the original function with `setTimeout`, 1 second later? Isn't this causing the Lambda Function to run for longer than needed?

Reply:
setTimeout code has been removed after implementing flush().